docs/user_guide.rst

```diff
@@ -134,8 +134,24 @@ identified by their indices in the quantum register, as you can see from the pri
 time you do not need to deal with IQM-style qubit names when using Qiskit, however when you need, the methods
 :meth:`.IQMBackend.qubit_name_to_index` and :meth:`.IQMBackend.index_to_qubit_name` can become handy.
 
+The results of a job, that was executed with IQM quantum computer, contain the original request with the
```

Comment:
Can you move this a bit up in the user guide. Just before the section `The ``backend`` instance ...`